deploy/docker/cp-dts-status-checker/README.md

```diff
@@ -0,0 +1,12 @@
+## This is a script for checking DTS statuses and sending notifications if they have been changed.
+### Prerequisites
+* Variables required for the script to work:
+
+| Env | Requirements | Example value | Description |
+| --- | --- | --- | --- |
+| CP_DTS_STATUS_NOTIFICATION_USER       | Is required | user1                   | The user who will receive notifications of DTS status change |
+| CP_DTS_STATUS_STORAGE                 | Is required | dts-status-storage      | The bucket where the data.json file with the previous DTS state will be saved. If the value is specified as in the example, it will look like "/cloud-data/dts-status-storage/data.json" in the system directory |
```

Comment:
as was discuss lets transform this variable into new one `CP_DTS_CHECKER_SYSTEM_DIR` 
it also should be required, and should be used to determine where previous status data is stored and where template.html is stored